tests/it/java_heap.py

```diff
@@ -18,17 +25,11 @@
 mb_total = scyjava.memory_total() // 1024 // 1024
 mb_used = scyjava.memory_used() // 1024 // 1024
 
-# Used memory should be less than the current memory total,
-# which should be less than the maximum heap size.
-assert mb_used <= mb_total <= mb_max, f"{mb_used=} {mb_total=} {mb_max=}"
-
-# The maximum heap size should be approximately 1 GB.
-assert 900 <= mb_max <= 1024, f"{mb_max=}"
+assert_that(mb_used, 'Used memory should be less than the current memory total').is_less_than_or_equal_to(mb_total)
+assert_that(mb_total, 'current memory total should be less than maximum memory').is_less_than_or_equal_to(mb_max)
+assert_that(mb_max, 'maximum heap size should be approx. 1 GB').is_between(900, 1024)
 
-# Most of that memory should still be free; i.e.,
-# we should not be using more than a few MB yet.
-assert mb_used <= 5, f"{mb_used=}"
+tolerance = pow(10, magnitude(mb_initial))
```

Comment:
Add a comment about what `tolerance` does here.

Reply:
This logic feels like overkill to me... I would have just changed the `5` to `10` and called it good :laughing:

Reply:
😂 you're not wrong, somehow felt better than just "10" - I'll change it to that

Reply:
I'll wait to add the comment on the above discussion, in test assertion frameworks they generally let you say how close two numbers should be to each other within some magnitude. This is that, but if y'all put the kibosh on assertpy then it would just be `< 10` (which I would still rename to tolerance or delta or something as opposed to an inlined number)

Reply:
We will keep `assertpy` but I also agree with @ctrueden that this does feel a bit much. I say just set the `tolerance` to `<10`.